yt/visualization/_handlers.py

```diff
@@ -61,13 +61,15 @@ class NormHandler:
         "_dynamic_range",
         "_norm_type",
         "_linthresh",
+        "_vcenter",
         "_norm_type",
```

Comment:
you didn't introduce this, but `"_norm_type"` is in the `__slots__` declaration twice... could remove one here?

Reply:
Good catch!  I removed it.